setup.py

```diff
@@ -1,38 +1,38 @@
 import os
 from setuptools import find_packages, setup
 
-with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+with open(os.path.join(os.path.dirname(__file__), "README.md")) as readme:
     README = readme.read()
 
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
 setup(
-    name='sentry_django_settings',
-    version='0.4',
+    name="sentry_django_settings",
+    version="0.5.0",
```

Comment:
Black changes. I'd normally not commit them, but it's all relatively harmless changes. The big change is right here, bumped up to 0.5.0